src/components/Currency/index.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+const Currency = props => {
+  const {
+    value,
+    withCurrencyLabel,
+    withSymbol,
+    currency,
+    fractionDigits,
+    ...rest
+  } = props;
+  const { t } = useTranslation();
+
+  const formattedCurrency = new Intl.NumberFormat(navigator.language, {
+    minimumFractionDigits: fractionDigits,
+    maximumFractionDigits: fractionDigits
+  }).format(value);
+
+  const getSymbol = () => {
+    switch (currency) {
+      case "AMD":
+        return "֏";
+      default:
```

Comment:
Having default to USD is a good idea, but we still need a case for USD explicitly.